inst/rmarkdown/templates/template-name/skeleton/skeleton.Rmd

```diff
@@ -1,21 +1,37 @@
 ---
-title: "The post title"
-date: "YYYY-MM-DDTHH:MM:SS"
+title: "The Post Title"
 output: hugodown::md_document
 status: "publish"
-slug: "the-post-slug"
```

Comment:
wait should this be kept? the users could still want to choose their own slug?

Reply:
Be default we will handle `slug` in `wp_post()` but users can still overwrite it if they manually provide `slug` in the YAML header. However, `slug` is not shown in the default template. What do you think?

Reply:
sorry for the delay, I think I'd like to keep it in the template because folks might wonder how to add it?